qemu/tests/cfg/blockdev_commit_reboot.cfg

```diff
@@ -0,0 +1,34 @@
+- blockdev_commit_reboot:
+    type = blockdev_commit_reboot
+    virt_test_type = qemu
+    only Linux
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
+    image_name_data = data
+    snapshot_tags = sn1 sn2 sn3 sn4
+
+    image_size_sn1 = 20G
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
```

Comment:
the format can be any besides qcow2?

Reply:
We only support qcow2 format snapshot.

Reply:
yes, as xutian explained, if the base is raw/qcow2, we just use qcow2 as snapshot format.